packages/lib/modules/pool/queries/usePoolEnrichWithOnChainData.tsx

```diff
@@ -14,9 +14,11 @@ import {
 } from '../../web3/contracts/abi/generated'
 import { isComposableStablePool } from '../pool.utils'
 import { cowAmmPoolAbi } from '../../web3/contracts/abi/cowAmmAbi'
-import { weightedPoolAbi_V3, vaultExtensionAbi_V3 } from '@balancer/sdk'
+import { vaultExtensionAbi_V3 } from '@balancer/sdk'
 import { getCompositionTokens } from '../pool-tokens.utils'
 
+const totalSupplyAbi = parseAbi(['function totalSupply() view returns (uint256)'])
```

Comment:
`weightedPoolAbi_V3` is not exposed by the SDK anymore so we now manually use the ABI function

Reply:
my bad we could have left this for you i didn't realize FE using it

Reply:
if there are v3 ABIs FE uses that could change with new deployments would be easy to add back into SDK automation as needed/desired

Reply:
No worries, it is ok to use `parseAbi`.